core/adapters/bridge.go

```diff
@@ -129,6 +136,7 @@ func baRunResultError(str string, err error) error {
 type bridgeOutgoing struct {
 	JobRunID    string      `json:"id"`
 	Data        models.JSON `json:"data"`
+	Meta        models.JSON `json:"meta"`
```

Comment:
Could the meta table be empty?  If so, maybe omitempty would be good here

Reply:
It's not null, so the 'emptiest' it can be is `{}`.